### DISCUSSION TITLE
Sysmenu display_seconds toggle issue

### DISCUSSION THREAD

**Discussion by camillew** [c3d/db48x#268] [General]
Hello,

In the system menu, the toggle for displaying seconds in the status bar is not displayed properly (It works correctly, only the X is incorrect.) It seems to come from a typo in the rendering:

```
    case MI_48STATUS_SECONDS:
        ln = flag_str(s, "Show seconds", Settings.show_time);           break;
```
where it should use `Settings.show_seconds`.

Thanks
Camille

**Comment by c3d** [comment]
Yes, I noticed that during the demo (issue #247). Did not have time to fix it yet.

**Comment by c3d** [comment]
Should be fixed with https://github.com/c3d/DB48X-on-DM42/commit/2d57b02a676d8721594a99b552218cdd9605dd94
